docs/tutorial/correlatedstacks.rst

```diff
@@ -32,11 +51,21 @@ You can obtain the image stack data as a `numpy` array using the `get_image()` m
 If the `channel` argument is not provided, the default behavior is `"rgb"` for 3-color images. For single-color
 images, this argument is ignored as there is only one channel available.
 
+
+Finally, the aligned image stack can also be exported to tiff format::
+
+    stack.export_tiff("aligned_stack.tiff")
+    stack[5:20].export_tiff("aligned_short_stack.tiff") # export a slice of the CorrelatedStack
+
+Correlating force with the image stack
+--------------------------------------
+
 Quite often, it is interesting to correlate events on the camera's to `channel` data.
 To quickly explore the correlation between images in a `CorrelatedStack` and channel data
 you can use the following function::
 
     # Making a plot where force is correlated to images in the stack.
+    stack = lk.CorrelatedStack("example.tiff)
     stack.plot_correlated(file.force1x)
 
 .. image:: correlatedstack.png
```

Comment:
If the Notebook kernel is interactive (eg having the line `%jupyter notebook `for Jupyter notebook, or `%jupyter widget `for jupyter lab), you can click on the left graph to select a particular force. The corresponding video frame will then automatically appear on the right.